### DISCUSSION TITLE
Can I disable phone number auth in default openlogin modal?

### DISCUSSION THREAD

**Discussion by antmanstar** [Web3Auth/Web3Auth#1284] [Help]
Please help me how I can do this.
Thanks

**Comment by arch1995** [comment]
Yes, you can disable phone auth by setting up `loginConfig` options.

For example:-

```
new Openlogin({
    loginConfig: {
        sms_passwordless: {
            showOnModal: false,
            showOnDesktop: false,
            showOnMobile: false,
        }
    }
})
```

**Reply by antmanstar** [reply]
thank you.

**Reply by namannndev** [reply]
SMS Login method is hidden, but if I type mobile number in text field & submit, it works..it sends SMS.
 Can I stop this behaviour?
